### ISSUE TITLE
Base58 script prefix & FoundersReward

### ISSUE THREAD

**Issue by sammy007** [zencashio/zen#23]
https://github.com/zencashio/zen/blob/master/src/chainparams.cpp#L119

```
        // guarantees the first 2 characters, when base58 encoded, are "zs"
        base58Prefixes[SCRIPT_ADDRESS]     = {0x20,0x96};
```

Does not match

```
vFoundersRewardAddress = {
            "t5DiT2y2b5dBSzr51dQSVS2KgsGHYCRY5Js", /* main-index: 0*/
```

What is wrong with it? I am guessing founders reward must start with "zs".

**Comment by sammy007** [comment]
No idea why you hard coded invalid address, but I am pretty sure split will never happen with it or I am missing something?

**Comment by sammy007** [comment]
Also, I used `zen-cli validateaddress zrH8KT8KUcpKKNBu3fjH4hA84jZBCawErqn` to get a scriptPubKey for one of your testnet tax addresses and the result is `a9149990975a435209031e247dccf9bc3e3ed3c813398720cbd4e07bb3f9924b0f7654dd152a62571f0a80babfb41a6d303b5a66690d000003cd1101b4`. I got a reject after submitting a block using this script. It only works with part of it `a9149990975a435209031e247dccf9bc3e3ed3c8133987` I have no idea why results are not consistent. Just curious why there is extra OPs added via `validateaddress`.
